pallets/anchors/src/common.rs

```diff
@@ -46,13 +51,16 @@ mod tests {
 	#[test]
 	fn test_get_days_since_epoch() {
 		// 1971-01-01  00:00:00
-		assert_eq!(get_days_since_epoch(31536000000), 365);
+		assert_eq!(get_days_since_epoch(31536000000), Some(365));
 
 		// 1971-01-01  00:00:01
-		assert_eq!(get_days_since_epoch(31536001000), 366);
+		assert_eq!(get_days_since_epoch(31536001000), Some(366));
 
 		// 1970-12-31  11:59:59
-		assert_eq!(get_days_since_epoch(31449600000), 364);
+		assert_eq!(get_days_since_epoch(31449600000), Some(364));
+
+		// Overflow Test with MAX u32 (after division)
+		assert_eq!(get_days_since_epoch(371085174374358017), None);
```

Comment:
```suggestion
		assert_eq!(get_days_since_epoch(u32::MAX), None);
```

Reply:
u32::MAX is `4294967295 ` which is smaller than the needed one that would trigger the overflow. This number `371085174374358017` is the minimum u64 needed to just go above the u32::MAX one.